### PR TITLE
[CWS] installation page improvements

### PR DESCRIPTION
**Pull request by paulcacheux**
### What does this PR do?

This PR:
- fixes the config filename
- adds a new tab for other host installs (same as debian/fedora but should be useful for customers trying to install on other supported platforms) 

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/security_platform/cloud_workload_security/getting_started.md [hunk 1]
```diff
@@ -116,7 +116,7 @@ docker run -d --name dd-agent \
 
 For a package-based deployment, the Datadog package has to be deployed: run `dkpg -i datadog-agent_7….deb`
 
-By default Runtime Security is disabled. To enable it, both the datadog.yaml and the system-probe.yaml files need to be adapted. Run the following commands to enable these configurations:
+By default Runtime Security is disabled. To enable it, both the security-agent.yaml and the system-probe.yaml files need to be adapted. Run the following commands to enable these configurations:
```
Comment (kayayarai) [inline content/en/security_platform/cloud_workload_security/getting_started.md:119]:
```suggestion
By default Runtime Security is disabled. To enable it, both the `security-agent.yaml` and the `system-probe.yaml` files need to be adapted. Run the following commands to enable these configurations:
```

File: content/en/security_platform/cloud_workload_security/getting_started.md [hunk 2]
```diff
@@ -142,6 +142,8 @@ Once you apply the changes, restart both the Security Agent and the system-probe
 
 For a package-based deployment, the Datadog package has to be deployed: run `yum/dnf install datadog-agent_7….rpm`
 
+By default Runtime Security is disabled. To enable it, both the security-agent.yaml and the system-probe.yaml files need to be adapted. Run the following commands to enable these configurations:
```
Comment (kayayarai) [inline content/en/security_platform/cloud_workload_security/getting_started.md:145]:
```suggestion
By default Runtime Security is disabled. To enable it, both the `security-agent.yaml` and the `system-probe.yaml` files need to be adapted. Run the following commands to enable these configurations:
```

File: content/en/security_platform/cloud_workload_security/getting_started.md [hunk 3]
```diff
@@ -160,6 +162,30 @@ echo "runtime_security_config.network.enabled: true" >> /etc/datadog-agent/syste
 [1]: https://app.datadoghq.com/account/settings#agent/kubernetes
 {{% /tab %}}
 
+{{% tab "Host (Others)" %}}
+
+For a package-based deployment, the Datadog package has to be deployed: install the package with your package manager.
```
Comment (kayayarai) [inline content/en/security_platform/cloud_workload_security/getting_started.md:167]:
```suggestion
For a package-based deployment, the Datadog package has to be deployed. Install the package with your package manager.
```

File: content/en/security_platform/cloud_workload_security/getting_started.md [hunk 4]
```diff
@@ -160,6 +162,30 @@ echo "runtime_security_config.network.enabled: true" >> /etc/datadog-agent/syste
...
+
+By default Runtime Security is disabled. To enable it, both the security-agent.yaml and the system-probe.yaml files need to be adapted. Run the following commands to enable these configurations:
```
Comment (kayayarai) [inline content/en/security_platform/cloud_workload_security/getting_started.md:169]:
```suggestion
By default Runtime Security is disabled. To enable it, both the `security-agent.yaml` and the `system-probe.yaml` files need to be adapted. Run the following commands to enable these configurations:
```

**Review by kayayarai — CHANGES_REQUESTED** [review]

**Review by kayayarai — APPROVED** [review]

**Comment by paulcacheux** [comment]
@kayayarai Thanks ! (for the changes and for the merge)

**Pull request merged**